src/ecpool_worker.erl

```diff
@@ -285,3 +294,62 @@ add_conn_callback(OnReconnect, undefined) ->
     [OnReconnect];
 add_conn_callback(OnReconnect, OldOnReconnect) ->
     [OnReconnect, OldOnReconnect].
+
+stop_supervisees(SubPids, Time) ->
+    lists:foreach(fun(Pid) ->
+        case try_stop_supervisee(Pid, Time) of
+            ok -> ok;
+            {error, killed} ->
+                logger:warning("[PoolWorker] supervisee ~p is force killed", [Pid]);
+            {error, Reason} ->
+                logger:warning("[PoolWorker] supervisee ~p terminated with error: ~p",
```

Comment:
no not log `normal` or `shutdown` or `{shutdown, _}` reasons.

Reply:
fixed